### ISSUE TITLE
logWriter_config.js example

### ISSUE THREAD

**Issue by tom-rosenfeld** [bluejamesbond/Scribe.js#62]
Seems like:
myLogWriter.getFilename = function (opt) {

should be:
myLogWriter.getFile = function (opt) {

**Comment by Kyle-Falconer** [comment]
I disagree: `getFile` sounds like the function should return either the contents of the file itself or a file handle or socket for writing to the file, and `getFilename` is asking specifically for the name of the file.

**Comment by tom-r-o** [comment]
If I recall correctly, I did not mean this as a suggestion for a function rename. This was a suggestion to fix a code sample that doesn't work (don't know if that's still the case). The code tries to invoke `getFilename`, while the object has a method named `getFile`.
